plugins/Login/SessionInitializer.php

```diff
@@ -42,61 +42,14 @@ class SessionInitializer
     private $usersManagerAPI;
 
     /**
-     * The authenticated session cookie's name. Defaults to the value of the `[General] login_cookie_name`
-     * INI config option.
-     *
-     * @var string
-     */
-    private $authCookieName;
-
-    /**
-     * The time in seconds before the authenticated session cookie expires. Only used if `$rememberMe`
-     * is true in the {@link initSession()} call.
-     *
-     * Defaults to the value of the `[General] login_cookie_expire` INI config option.
-     *
-     * @var string
-     */
-    private $authCookieValidTime;
-
-    /**
-     * The path for the authenticated session cookie. Defaults to the value of the `[General] login_cookie_path`
-     * INI config option.
-     *
-     * @var string
-     */
-    private $authCookiePath;
-
-    /**
-     * Constructor.
-     *
      * @param UsersManagerAPI|null $usersManagerAPI
-     * @param string|null $authCookieName
-     * @param int|null $authCookieValidTime
-     * @param string|null $authCookiePath
      */
-    public function __construct($usersManagerAPI = null, $authCookieName = null, $authCookieValidTime = null,
-                                $authCookiePath = null)
+    public function __construct($usersManagerAPI = null)
```

Comment:
BTW: We are using a custom auth cookie name in another plugin. Is it still possible to  use sessionInitializer with a custom authCookieName and the regular, configured authCookieName at the same time?

like in another plugin the user is authenticated using the regular cookie as normal. There may be a second authentication on top to access certain features which is currently solved by using like `new SessionInitializer(null, 'myCustomAuth')`

Reply:
As the class is API, removing these parameters be like basically a breaking change. BTW we also set a custom `$authCookieValidTime` value there.

Reply:
The auth cookie is no longer used, so this would be a problem. I can restore the old SessionInitializer, deprecate it and create an entirely new one for the new logic. Is that acceptable?

Reply:
That should work 👍  as long as everything still works